### ISSUE TITLE
TeamsChannelsPolicy fails export with MFA

### ISSUE THREAD

**Issue by Swampen** [microsoft/Microsoft365DSC#1053]
After trying to export the TeamsChannelsPolicy resource, it results in an empty configuration.
```powershell
 Export-M365DSCConfiguration -Quiet -ComponentsToExtract "TeamsChannelsPolicy" -Path . -GlobalAdminAccount $creds
[1/1] Extracting [TeamsChannelsPolicy]...⌛ Export took {3 seconds}
```
I encountered this on a fresh installed Windows 10 instance. 
After some investigation, I had to manually download the Skype for Business Online from Microsoft's website, since the `MSCloudLoginAssistant` module uses the `New-CsOnlineSession` cmdlet when MFA is required.Only then I am able to run an export once in that PowerShell instance. 
If I try to run the export again in the same instance, it goes back to export an empty configuration.

**Comment by Swampen** [comment]
The pull request https://github.com/microsoft/MSCloudLoginAssistant/pull/107 will fix this

**Comment by Swampen** [comment]
Solved in release 1.21.317.1
